packages/vscode-extension/lib/metro_helpers.js

```diff
@@ -64,13 +64,33 @@ function adaptMetroConfig(config) {
   // Handle the case when resolver is not defined in the config
   if (!config.resolver) {
     config.resolver = {};
+  } else {
+    const originalResolveRequest = config.resolver?.resolveRequest;
+    if (originalResolveRequest) {
+      // Override the resolveRequest method to include files in the ".storybook/"
+      // directory, which contains "storybook.requires.js".
```

Comment:
Add more descriptive comment here to explain why its here:

> Standard storybook setup relies on `resolveRequest` being overriden in order to exclude storybook files from bein imported into the bundle. The files are only included when STORYBOOK_ENABLED environment variable is set. Apparently, we can't set that variable for the whole metro process as you'd normally do with storybook, because it also controlls swapping out the main app entrypoint which also accesses that env constant via expo-constants module. We here implement a workaround which only sets the env variable for the duration of resolveRequest call and reset it back afterwards such that it only impacts resolution process.